easybuild/easyconfigs/r/ROCm/ROCm-4.2.0-GCC-10.3.0-amd.eb

```diff
@@ -0,0 +1,135 @@
+easyblock = 'Bundle'
+
+name = 'ROCm'
+version = '4.2.0'
+versionsuffix = '-amd'
+
+homepage = 'https://github.com/RadeonOpenCompute'
+description = "Platform for GPU Enabled HPC and UltraScale Computing"
+docurls = ['https://rocmdocs.amd.com']
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+builddependencies = [('CMake', '3.20.1')]
+
+dependencies = [
+    ('Mesa', '21.1.1'),  # required by ROCclr (for GL/glx.h)
+]
+
+# being a member of the 'video' group is required,
+# and need to run 'newgrp video' to have write access to /dev/kfd
+# cfr. https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html
+group = 'video'
+
+local_llvm_configopts = "-DCLANG_OPENMP_NVPTX_DEFAULT_ARCH=sm_70 -DLIBOMPTARGET_NVPTX_COMPUTE_CAPABILITIES=70 "
```

Comment:
Is this required for this EasyConfig? I think that until AMD combines AOMP into their "upstream" fork of LLVM, the version of LLVM built by this EC can't do AMD offloading. It then doesn't make that much sense to enable offloading to Nvidia. I'm asking because if we want to support Nvidia here we probably need an EasyBlock to handle `--cuda-compute-capabilities`